R/gh.R

```diff
@@ -208,6 +211,24 @@ gh <- function(endpoint, ..., per_page = NULL, .token = NULL, .destfile = NULL,
   res
 }
 
+gh_response_length <- function(res) {
+  if (!is.null(names(res)) && length(res) > 1 &&
+      names(res)[1] == "total_count") {
+    # Ignore total_count, incomplete_results, repository_selection
+    # and take the first list element to get the length
+    lst <- vapply(res, is.list, logical(1))
+    nm <- setdiff(
+      names(res),
+      c("total_count", "incomplete_results", "repository_selection")
```

Comment:
I assume these are coming from the API description?

Reply:
yeah